problem-spring-common/src/test/java/org/zalando/problem/spring/common/AdviceTraitLoggingTest.java

```diff
@@ -39,40 +30,35 @@ void reset() {
         TestLoggerFactory.clear();
     }
 
-    static Status[] data() {
-        return Status.values();
+    static HttpStatus[] data() {
```

Comment:
Why change it from Status to HttpStatus?

Reply:
That’s because the function being tested takes a HttpStatus as parameter